build.ps1

```diff
@@ -259,8 +259,13 @@ try
         # We must disable package testing here as the E2E csproj may reference new APIs that are not available in existing NuGet packages.
         $packageTempPath = $env:AZURE_IOT_LOCALPACKAGES
         $env:AZURE_IOT_LOCALPACKAGES = ""
+        
         # SDK binaries
         BuildProject . "Azure IoT C# SDK Solution"
+
+        # Samples
+        BuildProject security\tpm\samples "SecurityProvider for TPM Samples"
```

Comment:
We need to specify this project explicitly since it isn't a part of our sourcecode solution